bin/generate.py

```diff
@@ -190,15 +201,15 @@ def format_body(f):
 def format_header_arguments(f):
     s = []
     for arg in f.args:
-        s.append('{}: {}'.format(arg[0], translate_argument(*arg, f=f)))
+        s.append('{}: {}'.format(arg[0].lower(), translate_argument(*arg, f=f)))
```

Comment:
Why is this needed?

Reply:
I added tolower for all the arguments because there were too many warnings. I am also not sure if this is needed here as well. 

If you dislike the idea of tolower, then we should add the ignore.

Reply:
This is a good thing to lowercase. In this high-level crate, we want to see Rust-friendly names.

It actually made me realize that the logic in `generate.py` is broken, as it assumes everything is lowercase. I’ll push a fix.

Reply:
Thanks. And let me know if you need me to change something. Waiting for your publish to crates.io

Reply:
Done.